### PR TITLE
Update purchase flow styling

### PR DESCRIPTION
**Pull request by alex-deaconu**
## Description
Update purchase flow styling

Use madero-style
Update address and other form fields
Update validation messages
Update invoice and success wording

Fix issue with address dropdown fields empty state
and color

[stage-2]

## Motivation and Context
Billing Friction epic

## How Has This Been Tested?
Tested changes locally. Updated e2e tests.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
No

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc please review. Thanks!

File: web/scss/sections/checkout.scss [hunk 1]
```diff
@@ -1,3 +1,36 @@
+.purchase-app {
+  .btn-group-justified {
+    @extend .mb-0;
+
+    width: 400px;
```
Comment (ezequielc) [inline web/scss/sections/checkout.scss:5]: @alex-deaconu This fixed width is cropping the buttons a little bit on mobile. Maybe we need a conditional here. Thanks

**Review by ezequielc — APPROVED** [review]

File: web/scss/sections/checkout.scss (hunk 1 above)
Comment (alex-deaconu) [reply web/scss/sections/checkout.scss:5]: @ezequielc good catch, I will take a look and fix. Thanks!

Comment (alex-deaconu) [reply web/scss/sections/checkout.scss:5]: @ezequielc I've fixed by stacking the buttons for xs. Please review the last commit. Thanks!

**Comment by ezequielc** [comment]
@alex-deaconu Looks good 👍

**Pull request merged**